aiomysql/connection.py

```diff
@@ -465,6 +495,22 @@ def _connect(self):
             self._next_seq_id = 0
 
             yield from self._get_server_information()
+            if self._sslcontext and not self._unix_socket:
+                packet = make_auth_ssl(
+                        charset=charset_by_name(self._charset).id,
+                        client_flags=(self.client_flag ^ 2048))
+                self.write_packet(packet)
+                # upgrade connection to ssl
+                # close recent reader and write and keep socket connected
+                sock = self._writer._transport._sock
+                # patch asyncion
+                self._writer._transport._call_connection_lost = \
+                    types.MethodType(
+                        _call_connection_lost, self._writer._transport)
+                self._writer._transport._force_close(None)
```

Comment:
Is it possible to pass ssl context in https://github.com/aio-libs/aiomysql/pull/156/files#diff-b4987acea49665e9f985c0da9f5604f4R487  without upgrading socket ourself and let asyncio to handle this?